src/models/event.model.js

```diff
@@ -18,7 +18,7 @@ const ReccurencyTypes = {
 
 const eventSchema = new Schema({
     username: { type: String, required: true },
-    eventID: { type: String, required: true, unique: true, sparse: true },
+    eventID: { type: String, unique: true, sparse: true },
```

Comment:
Why is this field still here? Aren't we using _id for events now?

Reply:
we can open a new issue to remove this field.

Reply:
Noted, approved.